tutorial/chapter_2_9.md

```diff
@@ -8,6 +8,13 @@ with just one keystroke?
 
 `F5` to the rescue.
 
+Note: in some setups the shortcut may be mapped to `Ctrl + F5` without the Fn
```

Comment:
Is this documented somewhere? On all the systems (osx) where I tried it, I found it to be <kbd>F5</kbd> only. I didn't mention case sensitive sort with <kbd>Cmd</kbd> + <kbd>F5</kbd>  on purpose to save the learner from information overload.

Though there are instances where people claim that they had a keybinding similar to windows for performing sort command: http://stackoverflow.com/a/23578602/410367